### ISSUE TITLE
flake8-pyi: PYI015 false positive on TypeVar assignment

### ISSUE THREAD

**Issue by bluetech** [astral-sh/ruff#3860]
Given this pyi file:

```py
from typing import TypeVar
_T = TypeVar('_T')
```

running

```
ruff --select PYI015 --isolated --no-cache x.pyi
```

gives

```
x.pyi:2:6: PYI015 [*] Only simple default values allowed for assignments
```

I think `TypeVar` assignments should be excluded from this rule as they appear naturally in pyi files.

Possible workaround is `_T: TypeVar = TypeVar('_T')`, however that's unnecessarily redundant, and pyright doesn't like it.

Similar cases - `TypeAlias` (already handled), `NewType`, `TypeVarTuple`, `ParamSpec`

From a quick test, this doesn't happen on flake8-pyi.

I think I can provide a PR for this.

ruff 0.0.260
